packages/lzards-api-client/package.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
packages/lzards-api-client/README.md is empty

Reply:
un-emptied it.